.circleci/config.yml

```diff
@@ -1,18 +1,16 @@
 version: 2
 jobs:
-  build-and-test:
+  build-and-test-iOS-swift-4.2:
```

Comment:
I'd rename it to something shorter. They are all builds running tests. 
- `iOS-swift-4.2-build`
- `build-iOS-swift-4.2`

Reply:
Will think about it ;)